### PR TITLE
update: clarified the place where to put a dependency

### PR DESCRIPTION
**Pull request by PetrakovichVictoria**
https://youtrack.jetbrains.com/issue/KT-44911

### CODE REVIEW

File: docs/topics/mpp/mpp-add-dependencies.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [//]: # (title: Add dependencies)
 
 To add a dependency on a library, set the dependency of the required [type](gradle.md#dependency-types) (for example, `implementation`) in the 
-`dependencies` block of the source sets DSL.
+`dependencies` block in your [Gradle](gradle.md) build script.
```
Comment (koshachy) [inline docs/topics/mpp/mpp-add-dependencies.md:4]: TBH, I have a concern about `in` doubling in this sentence 🤔

Comment (PetrakovichVictoria) [reply docs/topics/mpp/mpp-add-dependencies.md:4]: Thank you, I will ask for a proofread.

Comment (p7nov) [reply docs/topics/mpp/mpp-add-dependencies.md:4]: Wouldn't this link be better here? https://kotlinlang.org/docs/mpp-dsl-reference.html#dependencies
It's MPP- and dependencies-specific, as opposed to just gradle.md.

Comment (PetrakovichVictoria) [reply docs/topics/mpp/mpp-add-dependencies.md]: Thank you, Pasha! I've got an idea – what if to leave the link to Gradle, but also add the link you proposed? See the result in the diff.

Regarding the proofread: there is a couple of minor changes, I put them in the code too, but proofreaders didn't have any concerns regarding doubling `in`.

**Review by koshachy — APPROVED** [review]
👍

**Review by p7nov — APPROVED** [review]
All good, thanks.

**Pull request merged**